test/vmtester/tests.cpp

```diff
@@ -75,11 +75,26 @@ TEST_F(evmc_vm_test, set_option_empty_value)
 {
     if (vm->set_option)
     {
-        enum evmc_set_option_result r = vm->set_option(vm, "unknown_option_csk9twq", nullptr);
+        evmc_set_option_result r = vm->set_option(vm, "unknown_option_csk9twq", nullptr);
         EXPECT_EQ(r, EVMC_SET_OPTION_INVALID_NAME);
     }
 }
 
+TEST_F(evmc_vm_test, set_option_unknown_value)
+{
+    auto r = evmc_set_option(vm, "verbose", "1");
+    if (r == EVMC_SET_OPTION_SUCCESS || r == EVMC_SET_OPTION_INVALID_VALUE)
```

Comment:
Perhaps add a comment here so that is clear: execute more tests if the `verbose` as an option name is valid.

That being said you may just want to to invert the condition to: `r != EVMC_SET_OPTION_INVALID_NAME`